src/experimental/safe_html_transform/tests/test_safe_html.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+import unittest2 as unittest
+from Products.PortalTransforms.data import datastream
```

Comment:
@prakharjoshi FYI: you are importing and testing the datastream method from Products.PortalTransforms here. This has nothing to do with your safe_html transform. You are testing a method from another package which does not really makes sense. Have a look at plone.intelligenttext how the transform is tested there...